front/src/applications/stdcm/components/StdcmForm/StdcmConsist.tsx

```diff
@@ -266,19 +263,21 @@ const StdcmConsist = ({
         />
       </div>
       <div className="loading-gauge">
-        <Select
-          id="loading-gauge-selector"
-          value={loadingGauge}
-          label={t('consist.loadingGauge')}
-          onChange={(e) => {
-            if (e) {
-              onLoadingGaugeChange(e);
-            }
-          }}
-          {...createStandardSelectOptions(GAUGE_LIST)}
-          disabled={disabled}
-          narrow
-        />
+        {!hideLoadingGaugeSelector && (
```

Comment:
We can do this and remove the `hideLoadingGaugeSelector`
```suggestion
        {(loadingGauges ?? []).length > 0 && (
```
or 
```suggestion
        {loadingGauge && loadingGauges.length > 0 && (
```

Reply:
Done